doc/api/errors.md

```diff
@@ -1669,6 +1669,13 @@ category.
 The `trace_events` module could not be loaded because Node.js was compiled with
 the `--without-v8-platform` flag.
 
+<a id="ERR_TRANSFERRING_EXTERNALIZED_SHAREDARRAYBUFFER"></a>
+### ERR_TRANSFERRING_EXTERNALIZED_SHAREDARRAYBUFFER
+
+A SharedArrayBuffer object was transfered but we do not see a lifetime partner 
+object and it was not us who externalized it - we are not sure how to serialize
+it because it's unclear how the memory is actually owned.
```

Comment:
This description makes sense as a code comment, but it might not be ideal for the docs. How about this?

> A `SharedArrayBuffer` object was encountered during serialization whose memory is not managed by the JavaScript engine or by Node. Such an `SharedArrayBuffer` can not be serialized.
>
> This can only happen when native addons create `SharedArrayBuffer`s in “externalized” mode, or put existing `SharedArrayBuffer` into externalized mode.

Reply:
Sure, sounds good.